torch/jit/__init__.py

```diff
@@ -1988,14 +1990,81 @@ def forward(self, input):
         """)
 
 
+def _unwrap_optional(x):
+    assert x is not None, "Unwrapping null optional"
+    return x
+
 _builtin_table = None
 
 _modules_containing_builtins = (torch, torch._C._nn)
 
-
-def _unwrap_optional(x):
-    assert x is not None, "Unwrapping null optional"
-    return x
+_builtin_ops = [
+    # Pairs of (function, op_name)
+    (_list_with_default, "aten::list_with_default"),
+    (_pair, "aten::_pair"),
+    (_quadruple, "aten::_quadruple"),
+    (_single, "aten::_single"),
+    (_triple, "aten::_triple"),
+    (_unwrap_optional, "aten::_unwrap_optional"),
+    (_wait, 'aten::wait'),
+    (cudnn.is_acceptable, "aten::cudnn_is_acceptable"),
+    (math.ceil, "aten::ceil"),
+    (math.copysign, "aten::copysign"),
+    (math.erf, "aten::erf"),
+    (math.erfc, "aten::erfc"),
+    (math.exp, "aten::exp"),
+    (math.expm1, "aten::expm1"),
+    (math.fabs, "aten::fabs"),
+    (math.floor, "aten::floor"),
+    (math.gamma, "aten::gamma"),
+    (math.lgamma, "aten::lgamma"),
+    (math.log, "aten::log"),
+    (math.log10, "aten::log10"),
+    (math.log1p, "aten::log1p"),
+    (math.pow, "aten::pow"),
+    (math.sqrt, "aten::sqrt"),
+    (math.isnan, "aten::isnan"),
+    (math.asinh, "aten::asinh"),
+    (math.atanh, "aten::atanh"),
+    (math.cosh, "aten::cosh"),
+    (math.sinh, "aten::sinh"),
+    (math.tanh, "aten::tanh"),
+    (math.acos, "aten::acos"),
+    (math.asin, "aten::asin"),
+    (math.atan, "aten::atan"),
+    (math.atan2, "aten::atan2"),
+    (math.cos, "aten::cos"),
+    (math.sin, "aten::sin"),
+    (math.tan, "aten::tan"),
+    (math.asinh, "aten::asinh"),
+    (math.atanh, "aten::atanh"),
+    (math.acosh, "aten::acosh"),
+    (math.sinh, "aten::sinh"),
+    (math.cosh, "aten::cosh"),
+    (math.tanh, "aten::tanh"),
+    (math.fmod, "aten::fmod"),
+    (math.modf, "aten::modf"),
+    (math.factorial, "aten::factorial"),
+    (math.frexp, "aten::frexp"),
+    (math.isnan, "aten::isnan"),
+    (math.isinf, "aten::isinf"),
+    (math.degrees, "aten::degrees"),
+    (math.radians, "aten::radians"),
+    (math.ldexp, "aten::ldexp"),
+    (torch._C._infer_size, "aten::_infer_size"),
+    (torch.nn.functional._no_grad_embedding_renorm_, "aten::_no_grad_embedding_renorm_"),
+    (torch.nn.functional.assert_int_or_pair, "aten::_assert_int_or_pair"),
+    (torch.nn.functional.interpolate, "aten::__interpolate"),
+    (torch.nn.functional.upsample_bilinear, "aten::__upsample_bilinear"),
+    (torch.nn.functional.upsample_nearest, "aten::__upsample_nearest"),
+    (torch.nn.functional.upsample, "aten::__upsample"),
+    (torch.nn.init._no_grad_fill_, "aten::_no_grad_fill_"),
+    (torch.nn.init._no_grad_normal_, "aten::_no_grad_normal_"),
+    (torch.nn.init._no_grad_uniform_, "aten::_no_grad_uniform_"),
+    (torch.nn.init._no_grad_zero_, "aten::_no_grad_zero_"),
+    (torch._C._get_tracing_state, "aten::_get_tracing_state"),
+    (warnings.warn, "aten::warn"),
```

Comment:
This won't be a full set of built in ops, there are also all of these https://github.com/pytorch/pytorch/blob/4d676d53a67e610689afbb34deabcac6ba8b8c50/torch/csrc/jit/script/compiler.cpp#L390

Reply:
Right now the builtins list throws an error when running so it isn't included in the docs at all, I think we can leave these for a follow up